R/add_cols_result_df_pd_changes.R

```diff
@@ -0,0 +1,61 @@
+#' Function that renames Merton model baseline result columns and adds additional
+#' columns for the second application of the model in the late & sudden scenario.
+#'
+#' @param data A dataframe that contains the result data from the baseline
+#'   Merton model
+#' @param horizon A cahracter vector of length 1 indicating whether to return
+#'   a result obejct for overall or yearly results. Must be "annual" or "overall"
+#'
+#' @family helper functions
+#'
+#' @export
+
+add_cols_result_df_pd_changes <- function(data,
```

Comment:
Style?

Reply:
wasn't the data argument supposed to be on the same line?